cypress/e2e/sauce_demo/login.cy.js

```diff
@@ -0,0 +1,124 @@
+describe('Test cases for Login page', () => {
+
+    beforeEach("load fixture", function () {
+        
+        cy.fixture("users").then((dataUsers)=>{
+            this.dataUsers=dataUsers;
+        })
+
+        cy.visit('/')
+        cy.hash().should('be.empty')
+    });
+    
+
+    it('test a correct user and a bad password', function () {
+        
+        cy.url().should('eq', 'https://www.saucedemo.com/')
+        
+        cy.get('#user-name').type(this.dataUsers.name_std)
+        cy.get('#password').type(this.dataUsers.badPass)
+        
+        cy.get('#login-button').click()
+
+        cy.get('#login_button_container > div > form > div.error-message-container.error > h3').should('contain','Epic sadface: Username and password do not match any user in this service')
+
+        cy.url().should('not.contain', 'https://www.saucedemo.com/inventory.html') 
+        
+    });
+
+    it('test a incorrect user and a good password', function () {
+
+        cy.url().should('eq', 'https://www.saucedemo.com/')
+        
+        cy.get('#user-name').type(this.dataUsers.name_fake)
+        cy.get('#password').type(this.dataUsers.pass)
+        
+        cy.get('#login-button').click()
+
+        cy.get('#login_button_container > div > form > div.error-message-container.error > h3').should('contain','Epic sadface: Username and password do not match any user in this service')
+
+        cy.url().should('not.contain', 'https://www.saucedemo.com/inventory.html') 
+
+    });
+
+    it('test a correct login for standard_user', function () {
```

Comment:
This test can also check for elements that are only on the page if the user login is successful

Reply:
I don't understand. I'm checking the user can successfully login the page, and also I'm verifying that user was redirected to home page. Are you asking for not to check the redirect thing or that should I check only login page?